WindowsServerDocs/administration/OpenSSH/OpenSSH_KeyManagement.md

```diff
@@ -145,19 +160,14 @@ After completing these steps, whenever a private key is needed for authenticatio
 To use the user key that was created above, the public key needs to be placed on the server into a text file called *authorized_keys* under users\username\\.ssh\\.
 The OpenSSH tools include scp, which is a secure file-transfer utility, to help with this.
 
-To move the contents of your public key (~\.ssh\id_ed25519.pub) into a text file called authorized_keys in ~\.ssh\ on your server/host.
-
-This example uses the Repair-AuthorizedKeyPermissions function in the OpenSSHUtils module which was previously installed on the host in the instructions above.
+To move the contents of your public key (~\.ssh\id_ed25519.pub) into a text file called authorized_keys in ~\.ssh\ on your server.
```

Comment:
Could you provide the list of supported algorithms?

Reply:
Hi @AkihiroSuda and sorry for the late reply, the supported algorithms are shown in the help text:
`usage: ssh-keygen [-q] [-b bits] [-t dsa | ecdsa | ed25519 | rsa]`


I did however update the section with recommended options:
https://github.com/MicrosoftDocs/windowsserverdocs/pull/4788/files#diff-e9b13ca0f3a6dcda51255be3272c0d6b2e11096a4bcc961a9b1d8c6b638d5568R87-R91